code/modules/clothing/masks/miscellaneous.dm

```diff
@@ -18,7 +20,42 @@
 /obj/item/clothing/mask/muzzle/attack_hand(mob/user as mob)
 	if(user.wear_mask == src && !user.IsAdvancedToolUser())
 		return 0
+	else if(user.wear_mask == src && security_lock && locked)
+		if(ishuman(user))
+			if(doUnlock(user))

```

Comment:
This won't break as it is only for unlock, locking occurs outside of this.

gags and tape both have a security_lock of 0